dashboard/static/css/file_access_within_week.css

```diff
@@ -1,5 +1,48 @@
 body {
-    font: 10px sans-serif;
+    font-size: 10px;
+    font-family: 'Open Sans', sans-serif;
+    font-weight: 400;
+    text-align: center;
+}
+
+#explanation {
+    font-size: 12px;
+    max-width: 620px;
+    margin: 0 auto;
+    padding-top: 10px;
+    color: #ababab;
+    font-weight: 300;
+}
+
+.y.axis line {
+    fill: none;
```

Comment:
I think CSS naming convention is having dashes(-) `.y.axis` is confusing as one can think it is referring to css 2 class name, how about `.y-axis` and `.x-axis`?